src/components/resource/_fields/coverage-statement/coverage-statement-fields.js

```diff
@@ -34,5 +63,9 @@ export function validate(values, { intl }) {
     });
   }
 
+  if (values.hasCoverageStatement === 'yes' && values.coverageStatement.length === 0) {
+    errors.coverageStatement = 'If selected, coverage statement cannot be blank.';
```

Comment:
This wording feels a little weird to me. We use `selected` in different ways and, if this gets read through a screenreader, that might be confusing. I don't have many suggestions on how to reword this, but maybe changing to, 
"Coverage statement field cannot be left blank if selected." might be clearer?

Reply:
Changed to "Statement field cannot be blank if selected."